src/config.c

```diff
@@ -1067,6 +1067,22 @@ void git_config_iterator_free(git_config_iterator *iter)
 	iter->free(iter);
 }
 
+int git_config_find_local(git_repository *repo, git_buf *path)
```

Comment:
Our "out" params are first in the decl, this would be better as:

    int git_config_find_local(git_buf *path, git_repository *repo);